improver/metadata/probabilistic.py

```diff
@@ -155,6 +155,37 @@ def find_threshold_coordinate(cube):
     return threshold_coord
 
 
+def probability_is_above_or_below(cube):
+    """Checks the spp__relative_to_threshold attribute and outputs
+    whether it is above or below the threshold given. If there isn't
+    a spp__relative_to_threshold attribute it returns None.
+
+    Args:
+        cube (iris.cube.Cube):
+            Cube containing thresholded probability data
+
+    Returns:
+        str:
+            Which indicates whether the cube has data that is
+            above or below the threshold
+    """
+
+    threshold_attribute = None
+    thresh_coord = find_threshold_coordinate(cube)
+    thresh = thresh_coord.attributes.get(
+        "spp__relative_to_threshold", None
+    )
+    if thresh in ("above", "greater_than",
+                   "greater_than_or_equal_to"):
+        threshold_attribute = "above"
+    elif thresh in ("below", "less_than", "less_than_or_equal_to"):
+        threshold_attribute = "below"
+    else:
+        threshold_attribute = None
```

Comment:
Don't need this line as you've already set `threshold_attribute = None` by default (at line 173).